android/src/main/java/com/twiliovoicereactnative/PreflightTestRecordDatabase.java

```diff
@@ -0,0 +1,37 @@
+package com.twiliovoicereactnative;
+
+import com.twilio.voice.PreflightTest;
+
+import java.util.UUID;
+
+public class PreflightTestRecordDatabase {
+  public static class PreflightTestRecord {
+    private final UUID uuid;
+
+    private PreflightTest preflightTest;
+
+    PreflightTestRecord(UUID uuid) {
+      this.uuid = uuid;
+    }
+
+    public UUID getUuid() {
+      return uuid;
+    }
+
+    public PreflightTest getPreflightTest() {
+      return preflightTest;
+    }
+  }
+
+  private PreflightTestRecord preflightTestRecord;
+
+  public void setRecord(UUID uuid, PreflightTest preflightTest) {
```

Comment:
there is only one, is the uuid necessary?

Reply:
Yeah, I was thinking about removing it, but we'll need it for when we emit the events back to the JS layer. I was thinking we could just have a PreflightTest event scope and not limit to UUIDs within that scope, but if we do that, then all the PreflightTest objects on the JS layer will get every event for all PreflightTest objects on the native layer. Which is unfortunately possible.